common/processor/processor/src/main/java/io/helidon/common/processor/ToAnnotationValueVisitor.java

```diff
@@ -21,93 +21,97 @@
 
 import javax.lang.model.element.AnnotationMirror;
 import javax.lang.model.element.AnnotationValue;
-import javax.lang.model.element.AnnotationValueVisitor;
 import javax.lang.model.element.VariableElement;
 import javax.lang.model.type.TypeMirror;
+import javax.lang.model.util.Elements;
 
-class ToStringAnnotationValueVisitor implements AnnotationValueVisitor<String, Object> {
+class ToAnnotationValueVisitor implements javax.lang.model.element.AnnotationValueVisitor<Object, Object> {
```

Comment:
Is this fully qualified package correct here? I mean, what is the reason for this change?

Reply:
Fixed, I renamed it to `AnnotationValueVisitor` which was incorrect, as it was in conflict with the interface name. Renamed, forgot to fix the FQ name. Original name was no longer correct